bin/posydon-run-pipeline

```diff
@@ -376,6 +376,11 @@ def train_interpolators(i, path_to_csv_file, verbose=False):
                                    and any(~np.isnan(grid.final_values[key]))
                                    and "MODEL" not in key)]
     
+    out_nan_keys = [key for key in grid.final_values.dtype.names
+                    if type(grid.final_values[key][0]) != np.str_
+                    and all(np.isnan(grid.final_values[key]))
+                ]
```

Comment:
Why did you exclude the check for "model_number" and "MODEL"? The first one is a MESA specific column to exclude the second one are columns from the supernova models, which should not be in there for the interpolator based on the mass transfer.

Reply:
fixed